packages/fxa-customs-server/lib/config/config.js

```diff
@@ -130,16 +130,18 @@ module.exports = function (fs, path, url, convict) {
       smsRateLimit: {
         limitIntervalSeconds: {
           doc: 'Duration of automatic throttling for sms',
-          default: 30 * 60,
+          default: 24 * 60 * 60, // One day
           format: 'nat',
           env: 'SMS_RATE_LIMIT_INTERVAL_SECONDS',
         },
         maxSms: {
           doc: 'Number of sms sent within rateLimitIntervalSeconds before throttling',
-          default: 3,
+          default: 9,
```

Comment:
I think we want to set this to `6` based on Slack conversation:
https://mozilla.slack.com/archives/C07T1GHL46S/p1736963075491979?thread_ts=1736960348.342549&cid=C07T1GHL46S

In theory there were notes to set the limit to 9/day for recovery phone setup, and 6/day for using the recovery phone during sign-in, but that might be overly complicated considering the current customs setup.

```suggestion
          default: 6,
```

Reply:
Ah gotcha, I didn't see the lower limit. I'll change the defualt.